scripts/src/generateMetaSchema.ts

```diff
@@ -22,11 +30,20 @@ console.log('Generating JSON Schema Meta-Schema...');
 const jsonSchema = zodToJsonSchema(Dictionary, {
 	name: 'Dictionary',
 	definitions: {
-		ReferenceTag,
-		ReferenceArray,
-		References,
+		SchemaBooleanField,
+		SchemaIntegerField,
+		SchemaNumberField,
+		SchemaStringField,
+		BooleanFieldRestrictions,
+		IntegerFieldRestrictions,
+		NumberFieldRestrictions,
+		StringFieldRestrictions,
+		ConditionalRestrictionTest,
 		Meta: DictionaryMeta,
 		Name: NameValue,
+		ReferenceArray,
+		ReferenceTag,
+		References,
 		Schema,
 		SchemaField,
 	},
```

Comment:
This list of schemas is the list of reusable definitions that the JSON Schema will be able to reference by name using the `$ref` syntax.